packages/app/src/components/AddressInput.tsx

```diff
@@ -0,0 +1,107 @@
+'use client'
+import { useState } from 'react'
+import { isAddress } from 'viem'
+import { normalize } from 'viem/ens'
+import { mainnet, sepolia } from 'wagmi/chains'
+import { createConfig, useEnsAddress, http, useEnsAvatar } from 'wagmi'
+import Image from 'next/image'
+
+interface AddressInputProps extends React.HTMLProps<HTMLInputElement> {
+  onRecipientChange: (address: string, isValid: boolean) => void
+  onRawInputChange?: (address: string) => void
+}
+// wagmi config
+export const config = createConfig({
```

Comment:
Not sure we need this. Is it because you want to specifically have a mainnet client for ENS? Either way, it probably shouldn't be on component level.

Reply:
Ah yeah it wasn't working on other networks without the config being explicitly defined to have the mainnet client. Will remove. 

With that change ENS will only be supported while connected to mainnet